presto-spi/src/main/java/com/facebook/presto/spi/statistics/StatisticsValue.java

```diff
@@ -14,16 +14,24 @@
 
 package com.facebook.presto.spi.statistics;
 
-import static java.util.Objects.requireNonNull;
-
-public final class StatisticsValue<T>
+public final class StatisticsValue
 {
-    private final DataFraction dataFraction;
-    private final T value;
+    private final double value;
+    private final double estimatorVariance;
```

Comment:
Unfortunately, `estimatorVariance` cannot be computed when using sample data (e.g. statistics) to compute population mean (e.g average row length). This is because variance for the entire population is unknown. In such case, T-student distribution might be used to estimate mean with given confidence level. Please have a read:
https://en.wikipedia.org/wiki/Student%27s_t-distribution#Sampling_distribution
https://en.wikipedia.org/wiki/Confidence_interval#Practical_example
https://en.wikipedia.org/wiki/Confidence_interval#Theoretical_example
https://en.wikipedia.org/wiki/Variance#Sample_variance
 
I suggest using confidence intervals to measure estimate "accuracy".